source/portable/NetworkInterface/pic32mzef/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ endif()
 #------------------------------------------------------------------------------
 add_library( freertos_plus_tcp_network_if STATIC )
 
-# TODO What about BufferAllocation_2.c here?
+# What about BufferAllocation_2.c here?
```

Comment:
I see some similar questions in the comments, are they needed?

Reply:
pic32mzef has its own propietary buffer management under same folder. I believe that's necessary for this platform.

Reply:
But unfortunately I can't verify it because I don't have this platform handy.